lib/models/movie.dart

```diff
@@ -0,0 +1,34 @@
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String overview;
+  final DateTime? releaseDate;
+  final String poster;
+  final String backDrop;
+  final double rate;
+  final List<String> genres;
+
+  Movie({
+    this.releaseDate,
+    required this.originalTitle,
+    required this.overview,
+    required this.title,
+    required this.backDrop,
+    required this.poster,
+    required this.rate,
+    required this.genres,
+  });
+
+
+  factory Movie.fromJson({required Map<String, dynamic> json, required List<String> genres, required String baseImageUrl}) {
```

Comment:
On the movie factory constructor you only need the json, not the genres and also not the baseImageUrl

Reply:
Done!